services/QuillLMS/app/controllers/teachers/progress_reports/activity_sessions_controller.rb

```diff
@@ -135,11 +135,12 @@ def return_data(should_return_json)
       csv_string = CSV.generate do |csv|
         csv << ['Student', 'Date', 'Activity', 'Score', 'Standard', 'Tool']
         activity_sessions.map do |session|
+          score = (session['percentage'].to_f == -1.0 ? 'Completed' : "#{session['percentage'].to_f*100}%")
```

Comment:
should be `===` instead of `==`

Reply:
This line is in Ruby not Javascript so I think the == syntax would be correct here.